su-template/default/module/html/method_summary.erb

```diff
@@ -0,0 +1,16 @@
+<% if method_listing.size > 0 %>
+  <% groups(method_listing) do |list, name| %>
+    <h2>
+      <a name="<%= name.gsub(' ', '_').downcase %>">
```

Comment:
> Any strong preference to the anchor symbol? I'm rather agnostic.

The Ruby Docs use the ¶ symbol, but I prefer the # symbol here.  It should have a title attribute with the text `"permalink"`.  Maybe the text "permalink" should be used instead of #, but I think just # is fine.